core/src/main/scala/org/apache/spark/scheduler/cluster/CoarseGrainedSchedulerBackend.scala

```diff
@@ -506,6 +522,21 @@ class CoarseGrainedSchedulerBackend(scheduler: TaskSchedulerImpl, val rpcEnv: Rp
       }
     }
 
+    conf.get(EXECUTOR_DECOMMISSION_CLEANUP_INTERVAL).map {
+      cleanup_interval =>
```

Comment:
If you don't mind, shall we move this to the upper line like the following?
```scala
conf.get(EXECUTOR_DECOMMISSION_CLEANUP_INTERVAL).map { cleanup_interval =>
```

Reply:
And, maybe, the variable name `cleanup_interval` looks like Python convention. Shall we renaming to `cleanupInterval` or `internal` simply?